src/ucs/stats/stats.h

```diff
@@ -19,10 +19,61 @@ BEGIN_C_DECLS
 
 /** @file stats.h */
 
+/* The maximum size of a class name */
+#define MAX_CLS_NAME_SIZE 50
```

Comment:
naming convention:
UCS_STATS_CLASS_NAME_MAX
same for other defines

Reply:
Fixed.